eventhub/Module.md

```diff
@@ -102,14 +80,14 @@ URI to the event hub namespace.
 Note:
 You must specify the SAS key name, SAS key and the resource URI when configuring the Azure Event Hub Client connector.
```

Comment:
Since we have mentioned before, we can remove this?

Reply:
Fixed in df7c75b . Thank you.